transaction-service/src/main/kotlin/ua/com/foxminded/restClient/nats/NatsController.kt

```diff
@@ -33,6 +33,7 @@ class NatsController @Autowired constructor(
 
 
     fun handleMessage(message: Message) {
+        println(message.data)
         val request = ProtoMessage.FindTransactionsByPersonIdAndTimeRequest.parseFrom(message.data)
         transactionService.findAllByIdAndBetweenDate(
```

Comment:
As I see from the name, this service will return many items as a result. and you publish a response from a subscriber, which means that on each received result response should be published. I believe it's not what is expected from this controller. And what if the service doesn't return anything?

Reply:
changed to return list